src/store/eventing/staged.ts

```diff
@@ -0,0 +1,7 @@
+import eventManager from '../../common/EventManager';
+import { AppState } from '../types';
+import { Event } from '../../@types';
+
+export const handleSetStagedEvents = (prevState: AppState, nextState: AppState): void => {
+    eventManager.emit(Event.STAGED_SET, nextState.creator.staged);
```

Comment:
Emit event to let Preview SDK know when `staged` is `null` and preview will focus on toolbar to prepare for  exiting the entire annotations mode.

Reply:
Can we use the `getStaged` selector here? Can we wrap the payload in an object so we can add to it later if needed: `{ staged: getStaged(nextState) }`